server.spec.js

```diff
@@ -0,0 +1,67 @@
+const request = require('supertest');
+
+const server = require('./server.js');
+
+describe('server.js', () => {
+    describe('root endpoint (/)', () => {
+        it('should return status code 200 OK', async () => {
+            const expected = 200;
+
+            const response = await request(server).get('/');
+
+            expect(response.status).toEqual(expected);
+        });
+
+        it('should return JSON', async () => {
+            const response = await request(server).get('/');
+
+            expect(response.type).toEqual('application/json');
```

Comment:
How did this one work out for you? There was some issue with it returning as an `text/html`...